Adds Android integration documentation (#304)
medic/cht-docs: content/en/apps/features/integrations/android.md

---
title: "Android"
weight: 1
description: >
   Launch the Android App with a link or from another app
relatedContent: >
  apps/reference/app-settings/token_login
---

The [CHT Android application](https://github.com/medic/medic-android/) can be launched by clicking on a link or invoking an intent in another Android app. This is useful for enabling login by SMS, directing a user to a specific page, and integrating between Android applications.

## Sending a URL

> When the user clicks on send the URL to your CHT instance in an SMS, email, Whatsapp, or any other app, Android will prompt the user to choose whether to open it in the Android app or the browser. If the app is not installed then the the URL will be opened in the browser.

Reviewer: Minor fixes:
```suggestion
When the user clicks on a link to a CHT instance from an SMS, email, WhatsApp, or any other app, Android will prompt the user to choose whether to open the URL in the Android app or the browser. If a CHT app is not installed then the URL will be opened in the browser.
```

Author: Whoa... I changed my mind so many times it ended up completely incomprehensible. Thanks!

![Android prompt](android-prompt.png "Android prompt screenshot")

> Users can choose "Always" to skip this prompt in future.

Reviewer: Great to include the screenshot. Added an additional note since the prompt can vary a fair bit between Android versions.

```suggestion
Users can choose "Always" to skip this prompt in future. The prompt may look different depending on the version of Android being used.
```
